### ISSUE TITLE
[UNVERIFIED] Update ownCloud library version

### ISSUE THREAD

**Issue by codinguser** [codinguser/gnucash-android#707]
[USER REPORT]
#### Steps to reproduce the behaviour
I've been testing out OwnCloud and NextCloud support, and I have a question. Exporting to OwnCloud appears to work, but NextCloud doesn't. I've set up OwnCloud and NextCloud to use the same username and password, and with OwnCloud, it says "OC server OK", "OC username/password OK", "Dir name OK". When I change the URL to the NextCloud install, it says "OC server OK", "Dir name OK", but "OC username/password invalid". Login from the web uses the same username and password for both accounts.

#### Expected behaviour
- The integration dialog should say "OC username/password OK"
Other clients (such as OwnCloud client for Linux desktop) work with NextCloud transparently. NextCloud even says they are a drop in replacement for OwnCloud, and all the apps should continue to work fine. Any ideas why only GnuCash would reject username or password?

#### Actual behaviour
Dialog says "OC username/password invalid"

#### Software specifications
* GnuCash Android version: 2.1.0
* System Android version: all
* Device type: all

**Comment by tribut** [comment]
Seeing the same behavior. From the access.log:

~~~~
"GET /status.php HTTP/1.1" 200 5196 "-" "Jakarta Commons-HttpClient"
"GET /index.php/ocs/cloud/user?format=json HTTP/1.1" 302 5069 "-" "Jakarta Commons-HttpClient"
~~~~

My guess is that Nextcloud wants to use a different URL and the App doesn't handle the redirect correctly.

**Comment by tribut** [comment]
Probably a duplicate of #651

**Comment by rivaldi8** [comment]
Fixed develop branch (fc261f7).
